mousebender/simple.py

```diff
@@ -42,6 +67,18 @@ def _normalize_project_url(url):
     return create_project_url(base_url, project_name)
 
 
+def _check_version(tag, attrs):
+    """Check if the tag is a PEP 629 tag and is a version that is supported."""
+    if tag != "meta" or attrs.get("name") != "pypi:repository-version":
+        return
+
+    major, minor = map(int, attrs["content"].split("."))
+    if major > _SUPPORTED_VERSION[0]:
+        raise UnsupportedVersion(major)
+    elif major == _SUPPORTED_VERSION[0] and minor > _SUPPORTED_VERSION[1]:
```

Comment:
This one too - the assumption is that things could only go higher. When the simple package version goes to 1.1, then it _might also go lower_ and while everything should be fine in that case, the intent here is to tell people when they are connecting to a packaging version that is unexpected (not higher and not lower).

Reply:
Going higher is fine as the PEP explicitly states that the expectation a newer minor version should work without issue.